bot/mc.go

```diff
@@ -0,0 +1,43 @@
+package bot
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"venova/sshd"
+
+	"github.com/gorcon/rcon"
+)
+
+func restartMinecraft() {
+	client, err := sshd.ConnectToDev()
+	if err != nil {
+		log.Printf("SSH error: %v", err)
```

Comment:
Return so we don't continue when we are in an invalid state

```suggestion
		log.Printf("SSH error: %v", err)
		return
```